src/main/resources/db/migration/V1_28__migrate_emails_to_new_structure.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TEMP TABLE tmp_email_addresses(id serial NOT NULL,prison_id VARCHAR(6)  NOT NULL,email_address varchar(100) NOT NULL, purpose_type varchar(40)  NOT NULL);
+
+INSERT INTO tmp_email_addresses(prison_id, email_address, purpose_type)
+    select prison_id, email_address, 'VIDEO_LINK_CONFERENCING'
+    from videolink_conferencing_centre GROUP BY prison_id, email_address;
+
+INSERT INTO tmp_email_addresses(prison_id, email_address, purpose_type)
+    select prison_id, email_address, 'OFFENDER_MANAGEMENT_UNIT'
+    from offender_management_unit GROUP BY prison_id, email_address;
+
+INSERT INTO email_address(value)
+    select email_address
+    from tmp_email_addresses GROUP BY email_address;
+
+INSERT INTO contact_details(prison_id, purpose_type)
+    select prison_id, purpose_type
+    from tmp_email_addresses ORDER BY ID;
+
+INSERT INTO contact_details_to_email_address(contact_details_id,email_address_id)
+            select sb.cd_id,sb.ea_id from (select cd.id as cd_id,ea.id as ea_id from tmp_email_addresses tmp
+                                                      join contact_details cd on cd.prison_id=tmp.prison_id and cd.purpose_type=tmp.purpose_type
+                                                      join email_address ea on ea.value = tmp.email_address
+                                            ) as sb;
+
+DROP TABLE offender_management_unit, videolink_conferencing_centre;
```

Comment:
should we remove this from the script for now and add it at a later date when we confirm that all data has been migrated?

Reply:
This will be tested in DEV, and other non prod envs, I have also tested this manually

Reply:
Sure its just that the script will be run on release and not manually makes me nervous. Maybe its just me :)

Reply:
I could put it in a follow up release, the drop part any how would that work @danbenton-mojdt and @dhirajshettymoj ?